src/mbgl/style/expression/coercion.cpp

```diff
@@ -25,7 +25,7 @@ EvaluationResult toNumber(const Value& v) {
         [](const std::string& s) -> optional<double> {
             try {
                 return util::stof(s);
-            } catch (const std::exception&) {
```

Comment:
Is this needed? I tested it here and worked fine everywhere:
https://github.com/mapbox/mapbox-gl-native/commit/f10d4febf70e309efff2d836c3256e36573d6585

Reply:
Yes - otherwise it breaks macOS build :disappointed: it looks like Apple's clang libc++ is broken.